libvast/src/system/indexer.cpp

```diff
@@ -63,13 +63,18 @@ indexer(caf::stateful_actor<indexer_state>* self, path filename,
     return {};
   }
   auto handle_batch = [=](const std::vector<table_slice_column>& xs) {
+    VAST_DEBUG(self, self->state.col.filename().str(), "got batch with",
+               xs.size(), "slices; starting at", xs.begin()->slice->offset(),
+               "; ending at:", xs.rbegin()->slice->offset());
     auto t = atomic_timer::start(*self->state.measurement);
     auto events = uint64_t{0};
     for (auto& x : xs) {
       events += x.slice->rows();
       self->state.col.add(x);
     }
     t.stop(events);
+    VAST_DEBUG(self, self->state.col.filename().str(), "added", events,
+               "events");
```

Comment:
```suggestion
```

Reply:
Both of these statements have been removed from the change that originally introduced them.